common/djangoapps/util/views.py

```diff
@@ -271,6 +279,12 @@ def _record_feedback_in_zendesk(
             "tags": zendesk_tags
         }
     }
+
+    # TODO: Determine whether or not the custom fields should be set like this or in a private
+    # update, as is done with additional_info beginning on line 309.
```

Comment:
It appears that this can be done from within Zendesk. The admin can choose which custom fields are visible to the end user, so we shouldn't need to worry about it here.